contracts/interchain-token-service/src/contract.rs

```diff
@@ -186,6 +188,60 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
             .into()
     }
 
+    fn interchain_token_id(env: &Env, sender: Address, salt: BytesN<32>) -> BytesN<32> {
+        env.crypto()
+            .keccak256(&(PREFIX_INTERCHAIN_TOKEN_ID, sender, salt).to_xdr(env))
+            .into()
+    }
+
+    fn deploy_interchain_token(
```

Comment:
```suggestion
    fn deploy_interchain_token_with_minter(
```
Add a separate `deploy_interchain_token` method that passes minter as `None` instead

Reply:
You can also modify this PR to not include additional minter support for now, and add that in a follow up